test/integration/uk/gov/hmrc/exports/repositories/SubmissionRepositorySpec.scala

```diff
@@ -87,6 +87,12 @@ class SubmissionRepositorySpec
         submissionsInDB.head must equal(submission)
       }
     }
+
+    "allow save two submissions with empty actions" in {
```

Comment:
Please can you prove this index works with a negative test as well
e.g. add two actions with the same ID and expect an exception

Reply:
Covered at line 68 in `trying to save Submission with the same conversationId twice throw DatabaseException`